### PR TITLE
Create membersandsettings.png

### PR DESCRIPTION
**Pull request by msbemba**
Added a new Screenshot as the navigations have changed.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Fixes# https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10785

**Comment by msbemba** [comment]
Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10796

**Comment by msbemba** [comment]
This one fixes two issues, created one PR as it is the same document.

File: microsoft-365/compliance/ediscovery-teams-workflow.md [hunk 1]
```diff
@@ -72,19 +72,20 @@ The first step to managing Teams content in eDiscovery (Premium) is to create a
...
-To create a case:
-
 1. Go to <https://compliance.microsoft.com> and sign in.
 
 2. In the left navigation pane of the Microsoft Purview compliance portal, click **eDiscovery > Premium**.
 
 3. On the **eDiscovery (Premium)** page, click the **Cases** tab, and then click **Create a case**.
 
-   The **New eDiscovery case** flyout page is displayed. The **Case format** section provides the option to create a case using the new case format.
+   The **New eDiscovery case** flyout page is displayed. The **Case format** section provides the option to create a case using the new case format, Click **Next** to proceed to the **Members and settings** Page.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-teams-workflow.md]:
```suggestion
   The **New eDiscovery case** flyout page is displayed. The **Case format** section provides the option to create a case using the new case format. Select **Next** to proceed to the **Members and settings** page.
```

File: microsoft-365/compliance/ediscovery-teams-workflow.md [hunk 2]
```diff
@@ -72,19 +72,20 @@ The first step to managing Teams content in eDiscovery (Premium) is to create a
...
 
    ![Large case option on the New eDiscovery case page.](..\media\AeDNewCaseFormat1.png)
+   
+ 4. On the **Members and Settings** Page, Add team members and configure settings and Click on **next** to proceed to the **Summary** page
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-teams-workflow.md]:
```suggestion
 4. On the **Members and Settings** page, add team members and configure settings, then select **next** to proceed to the **Summary** page.
```

File: microsoft-365/compliance/ediscovery-teams-workflow.md [hunk 3]
```diff
@@ -72,19 +72,20 @@ The first step to managing Teams content in eDiscovery (Premium) is to create a
...
+   ![Members and settings page](..\media\membersandsettings.png
 
-4. After naming the case, select the **New** option, and then click **Save** to create the case.
+4. On the Summary of the Case, Review the settings and click **Submit** to create the case.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-teams-workflow.md]:
```suggestion
4. In the summary of the case, review the settings and select **Submit** to create the case.
```

File: microsoft-365/compliance/ediscovery-teams-workflow.md [hunk 4]
```diff
@@ -167,9 +168,7 @@ To create a collection of Teams content:
...
 
-   To help ensure the most comprehensive collection of Teams chat conversations (including 1:1, group, and channel chats) use the **Type** condition and select the **Instant messages** option. We also recommend including a date range or several keywords to narrow the scope of the collection to items relevant to your investigation. Here's a screenshot of a sample query using the **Type** and **Date** options:
-
-   ![Query to collect Teams content.](..\media\TeamsConditionsQueryType.png)
+   To help ensure the most comprehensive collection of Teams chat conversations (including 1:1, group, and channel chats) use the **Message Kind** condition and select **Equals Any of** option and type **Microsoftteams**. We also recommend including a date range or several keywords to narrow the scope of the collection to items relevant to your investigation. Here's a screenshot of a sample query using the **Type** and **Date** options:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery-teams-workflow.md]:
```suggestion
   To help ensure the most comprehensive collection of Teams chat conversations (including 1:1, group, and channel chats), use the **Message Kind** condition, select the **Equals any of** option, and enter **Microsoftteams**. We also recommend including a date range or several keywords to narrow the scope of the collection to items relevant to your investigation. Here's a screenshot of a sample query using the **Type** and **Date** options:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder.

**Comment by yogkumgit** [comment]
Closing as per the comment in #10785

**Pull request closed without merging**